packages/react-router-with-zod/tests/index.test-d.ts

```diff
@@ -0,0 +1,98 @@
+import {
```

Comment:
We should add some runtime unit tests (in addition to the type tests here) to confirm that the validator helper is functioning as intended.